examples/agent-examples.yaml

```diff
@@ -0,0 +1,71 @@
+# Example: Agents and Agent Relationships
+
+'@context':
+  '@vocab': https://w3id.org/valueflows#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#,
+  alice: https://alice.example/
```

Comment:
Instead of three lines above we can use 
```yaml
  - https://git.io/vf-examples-jsonld-context
  - alice: https://alice.example/
```
lines below just need to align bob and alice, this way we have array with two elements, IRI to external context and object with local prefixes for agents

Reply:
ok - so we would need to make one context that all examples could use, right?

Reply:
yes, I made https://git.io/vf-examples-jsonld-context to use it in all examples and keep them consistent

Reply:
please note that first two elements need `-` since we create an array here